test/io/readfile/Reading_from_pipe.chpl

```diff
@@ -0,0 +1,14 @@
+use Subprocess;
+use IO;
+var sub = spawn(["echo","How are you?"],stdout=PIPE);
+
+var line:bytes;
+var st:iostyle;
+
+sub.stdout._set_style(style = new iostyle(binary=1));
+while sub.stdout.readbytes(line) {
```

Comment:
The while loop should not be needed since `readbytes` reads until EOF, right?

Reply:
@mppf  yes, I shall remove it.